### PR TITLE
Added platform check

### PR DESCRIPTION
**Pull request by Kwaizer**
Resolves: https://github.com/AlmaLinux/build-system/issues/213

### CODE REVIEW

**Comment by Kwaizer** [comment]
Do I need to fix syntax in the same pull request?

File: alws/crud/errata.py [hunk 1]
```diff
@@ -140,6 +140,10 @@ async def get_oval_xml(
         select(models.Platform).where(models.Platform.name == platform_name)
     )
     platform: models.Platform = platform.scalars().first()
+
+    if platform is None:
+        return None
```
Comment (javihernandez) [inline alws/crud/errata.py:145]:
```
if not platform:
    return
```

File: alws/routers/errata.py [hunk 2]
```diff
@@ -61,7 +61,13 @@ async def get_oval_xml(
     only_released: bool = False,
     db: AsyncSession = Depends(get_db),
 ):
-    return await errata_crud.get_oval_xml(db, platform_name, only_released)
+    records = await errata_crud.get_oval_xml(db, platform_name, only_released)
+    if records is None:
```
Comment (javihernandez) [inline alws/routers/errata.py:65]: here, we can also have `if not records`

**Comment by maccelf** [comment]
> Do I need to fix syntax in the same pull request?

Yes

**Review by Korulag — APPROVED** [review]

**Pull request merged**